File: Selenium/Project/activity3.py
# Get the copyright text

from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("http://alchemy.hguy.co/crm")

footerText = driver.find_element(By.ID, "admin_options").text
print(footerText)

assert footerText == "© Supercharged by SuiteCRM"
driver.quit()

File: Selenium/Project/activity3.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.junit.jupiter.api.Assertions;

//Get the copyright text

public class activity3 {
    public static void main(String[] args) throws InterruptedException {
        // Install driver
        WebDriverManager.firefoxdriver().setup();

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "NULL");
        WebDriver driver = new FirefoxDriver();

        driver.get("http://alchemy.hguy.co/crm");
        String footerText = driver.findElement(By.id("admin_options")).getText();
        System.out.println(footerText);
        Assertions.assertEquals("© Supercharged by SuiteCRM", footerText);

        driver.quit();
    }
}